containers/go-postgres/.devcontainer/.env

```diff
@@ -0,0 +1,4 @@
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=postgres
+POSTGRES_DB=postgres
+POSTGRES_HOSTNAME=postgresdb
```

Comment:
What do you mean?

Reply:
Ah, I understand what you mean. I would argue that since these containers are meant to be plug and play, and run immediately upon being downloaded, that would be the wrong choice. We want there to be as little gap between the user downloading and running the container as possible.